<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SMTPForm</class>
 <widget class="QWidget" name="SMTPForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>799</width>
    <height>457</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SMTP client</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="PersonalDataBox">
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="title">
      <string>SMTP</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="1" column="2">
       <layout class="QFormLayout" name="formLayout">
        <item row="1" column="0">
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout1">
            <item>
             <widget class="QLabel" name="labelUser">
              <property name="text">
               <string>Username:</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout2">
            <item>
             <widget class="QLabel" name="labelPassword">
              <property name="text">
               <string>Password:</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout3">
            <item>
             <widget class="QLabel" name="labelSendTo">
              <property name="text">
               <string>Send to:</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout4">
            <item>
             <widget class="QLabel" name="labelFolder">
              <property name="text">
               <string>File:</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout5">
            <item>
             <widget class="QLabel" name="labelKeyword">
              <property name="text">
               <string>Keyword:</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item row="1" column="1">
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_1">
            <item>
             <widget class="QLineEdit" name="lineEditUser"/>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QLineEdit" name="lineEditPassword">
              <property name="echoMode">
               <enum>QLineEdit::Password</enum>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QLineEdit" name="lineEditSendTo"/>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <widget class="QLineEdit" name="lineEditFolder"/>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_5">
            <item>
             <widget class="QLineEdit" name="lineEditKeyword"/>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item row="3" column="1">
         <widget class="QPushButton" name="btnSend">
          <property name="font">
           <font>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Try to Send</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QTextEdit" name="textOutput"/>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="labelOutput">
          <property name="text">
           <string>Output:</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
